pkg/api/validation/validation.go

```diff
@@ -687,11 +687,10 @@ func ValidatePodStatusUpdate(newPod, oldPod *api.Pod) errs.ValidationErrorList {
 
 	allErrs = append(allErrs, ValidateObjectMetaUpdate(&oldPod.ObjectMeta, &newPod.ObjectMeta).Prefix("metadata")...)
 
-	// TODO: allow change when bindings are properly decoupled from pods
```

Comment:
Why this change? I'm not particularly familiar with this code, but I figured you had a reason.